kornia/feature/loftr/loftr.py

```diff
@@ -119,20 +119,20 @@ def forward(self, data: dict) -> Dict[str, torch.Tensor]:
         """
 
         # 1. Local Feature CNN
-        data.update({
-            'bs': data['image0'].size(0),
-            'hw0_i': data['image0'].shape[2:], 'hw1_i': data['image1'].shape[2:]
-        })
+        bs = torch.tensor(data['image0'].size(0))
```

Comment:
```suggestion
        bs = torch.as_tensor(data['image0'].size(0))
```
to avoid copy

Reply:
are shapes really required to be converted to tensors ? maybe the dict can support `Any` ? /cc @ducha-aiki

Reply:
i believe dict can support `Any` but then you end up possibly duplicating code as you'd have to run a condition for jit scripting like so:
`if torch.jit.isinstance(x, Dict[str, torch.Tensor]):`
not sure if there is a workaround.